src/openrct2/ride/ride_ratings.c

```diff
@@ -610,32 +610,71 @@ static void ride_ratings_calculate(rct_ride *ride)
 
 static void ride_ratings_calculate_value(rct_ride *ride)
 {
+    static const int age_table_new[][4] = {
+        //Monhts    Multiplier  Divisor     Summand
```

Comment:
`Months`

I think it might simplify the logic if you did it the other way round start at the minimum age.